cmd/kubelet/app/server.go

```diff
@@ -146,7 +148,7 @@ type KubeletServer struct {
 type KubeletBootstrap interface {
 	BirthCry()
 	StartGarbageCollection()
-	ListenAndServe(net.IP, uint, *kubelet.TLSOptions, bool)
+	ListenAndServe(net.IP, uint, *kubelet.TLSOptions, kubelet.AuthInterface, bool)
```

Comment:
Predates this PR, but why are we using anonymous arguments for the second and fifth?  Seems like they need names.